l10n_it_website_portal_fiscalcode/views/l10n_it_website_portal_fiscalcode_templates.xml

```diff
@@ -7,19 +7,15 @@
     <template
         id="fiscalcode_details"
         name="Add fiscalcode"
-        inherit_id="portal.portal_my_details"
+        inherit_id="portal.portal_my_details_fields"
     >
-        <xpath
-            expr="//div[hasclass('o_portal_details')]/div[1]/div[1]/div[7]"
-            position="after"
-        >
-            <!-- After VAT -->
+        <xpath expr="//div[label[@for='vat']]" position="after">
             <div
                 t-attf-class="form-group #{error.get('fiscalcode') and 'o_has_error' or ''} col-xl-6"
             >
                 <label
                     class="col-form-label label-optional"
-                    for="vat"
```

Comment:
[riga 29](https://github.com/OCA/l10n-italy/pull/3338/commits/1f60375bf6571e128bd9d29bff51a3e09364478c#diff-743441041902fd9270de6909869d5f66cdf8c9bbc2642c2a864dffd9b24cc45cL28)

t-att-disabled="None if partner_can_edit_vat else '1'"

[riga 30](https://github.com/OCA/l10n-italy/pull/3338/commits/1f60375bf6571e128bd9d29bff51a3e09364478c#diff-743441041902fd9270de6909869d5f66cdf8c9bbc2642c2a864dffd9b24cc45cL30)

t-att-title="None if partner_can_edit_vat else 'Changing Fiscal Code is not allowed once document(s) have been issued for your account. Please contact us directly for this operation.'"

Reply:
done!

Reply:
https://github.com/odoo/odoo/blob/16.0/addons/website_sale/views/templates.xml#L1843

Per consistenza con gli altri campi, sarei per mantenere `readonly` invece che `disabled`

Reply:
ti chiedevo di modificarla perché coerente con la vista 
https://github.com/odoo/odoo/blob/16.0/addons/portal/views/portal_templates.xml#L404 
che stai facendo modificando

Reply:
> done!

puoi controllare vedo ancora `if partner.partner.can_edit_vat() else` invece che `if partner_can_edit_vat else`

Reply:
Lo sapevate che se non salvi il file, poi sul file non vengono scritte le modifiche? Sapevatelo.